package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@readme/markdown",
+  "name": "@readme/mdx",
```

Comment:
We going to be publishing under this name now? Or have we been doing that all along?

Reply:
Oh shoot, good catch. That's to get npm link to work in the main app